packages/ui/src/forms/Label/index.tsx

```diff
@@ -5,10 +5,15 @@ import { getTranslation } from '@payloadcms/translations'
 import React from 'react'
 
 import { useTranslation } from '../../providers/Translation/index.js'
+import { useFieldPath } from '../FieldPathProvider/index.js'
+import { useForm } from '../Form/context.js'
 import './index.scss'
 
 const Label: React.FC<LabelProps> = (props) => {
-  const { htmlFor, label, required = false } = props
+  const { htmlFor: htmlForFromProps, label, required = false } = props
+  const { uuid } = useForm()
+  const { path } = useFieldPath()
+  const htmlFor = htmlForFromProps || `field-${path?.replace(/\./g, '__')}${uuid ? `-${uuid}` : ''}`
```

Comment:
I want you to consolidate the formatting of this value. It is duplicative across a number of files. We can export from one central location and use it everywhere.

Reply:
@PatrikKozak good call, on it.

Reply:
@jacobsfletch All set.